cmd/thanos/query.go

```diff
@@ -146,6 +146,8 @@ func registerQuery(app *extkingpin.App) {
 	enableMetricMetadataPartialResponse := cmd.Flag("metric-metadata.partial-response", "Enable partial response for metric metadata endpoint. --no-metric-metadata.partial-response for disabling.").
 		Hidden().Default("true").Bool()
 
+	featureList := cmd.Flag("experimental", "Comma separated experimental feature names to enable.").Default("").Strings()
```

Comment:
let's use the same `--enable-feature` as Prometheus? Also please list the available options.